src/Knp/DictionaryBundle/Dictionary/CallableDictionary.php

```diff
@@ -116,11 +120,9 @@ public function getIterator()
      */
     public function serialize()
     {
-        $this->hydrate();
-
         return serialize([
             'name'   => $this->name,
-            'values' => $this->values,
+            'values' => $this->values ?: array(),
```

Comment:
It was fix by `php-cs-fixer`. PHP version in composer.json should be bumped, as well as `.php_cs` should be updated accordingly. But this is not the purpose of this PR ;)